### PR TITLE
DOC-10614 Java txns from Scala

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

**Comment by RichardSmedley** [comment]
@programmatix   Let me know how we can improve this - and if it would be fine in Kotlin, too.

**Review by programmatix — APPROVED** [review]

**Comment by programmatix** [comment]
Yes I think it would broadly be fine in Kotlin, but the Scala code snippet may need a little tweaking of course.
Double-checking with @dnault

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 1]
```diff
@@ -9,5 +9,28 @@ include::project-docs:partial$attributes.adoc[]
 {description}
 
 
-Distributed ACID Transactions are not currently available for the Scala SDK.
-include::{version-common}@sdk:shared:partial$acid-transactions.adoc[tag=other]
+Distributed ACID Transactions are implemented in the Java SDK, and simple to integrate into your Scala client code.
+
+
+== Java Transactions from the Scala SDK
```
Comment (dnault) [inline modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc:15]: Yes, I think we could say "from the Scala or Kotlin SDK", and "in your Scala or Kotiln app" down below.

Might also be worth mentioning the user needs to include the Couchbase Java SDK in their project.

Comment (dnault) [reply modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc:15]: Sorry, I missed that this page is specifically for `docs-sdk-scala`. s/Scala/Kotlin would work for `docs-sdk-kotlin`.

**Pull request merged**